eclair-core/src/main/scala/fr/acinq/eclair/channel/fsm/DualFundingHandlers.scala

```diff
@@ -59,25 +59,34 @@ trait DualFundingHandlers extends CommonFundingHandlers {
     }
   }
 
+  def pruneCommitments(w: WatchFundingConfirmedTriggered, d: DATA_WAIT_FOR_DUAL_FUNDING_CONFIRMED): Try[DualFundingTx] = {
```

Comment:
Why return a `Try`? This can simply be an `Option[DualFundingTx]`.

Reply:
I had to have a `Throwable` to feed to `handleLocalError`, but that was before https://github.com/ACINQ/eclair/pull/2557#discussion_r1063599710.

Reply:
Done as part of [ff5aada](https://github.com/ACINQ/eclair/pull/2557/commits/ff5aada6abce62cb8081400bbf8666577c890fd3).